.github/workflows/ci-owasp-dependency-check.yaml

```diff
@@ -30,10 +30,22 @@ jobs:
     name: Run OWASP Dependency Check
```

Comment:
One small modification, it's worth adding `${{ matrix.name }}` are part of the name so that the jobs can be identified in the GitHub Actions UI and reports.

Reply:
example: https://github.com/apache/pulsar/blob/4d3a52048bcd1cbb73c6f219be015ccba3110fe0/.github/workflows/ci-maven-cache-update.yaml#L49